frontend/src/components/molecules/feed/UserCard.tsx

```diff
@@ -25,7 +25,7 @@ interface UserCardProps {
 
 const UserCard = (props: UserCardProps) => {
```

Comment:
the rating diff must move a little bit to the right/left
![image](https://github.com/nerraou/ft_transcendence/assets/28894990/1c8156ca-0ea7-4fd6-ad28-d9d9b9251567)